ocrd/ocrd/processor/base.py

```diff
@@ -104,7 +104,7 @@ def __init__(
                 print(res)
             return
         if show_resource:
-            res_fname = list_resource_candidates(ocrd_tool['executable'], show_resource, is_file=True)
+            res_fname = list_resource_candidates(ocrd_tool['executable'], show_resource)
```

Comment:
This does not work, though. Just removing `is_file` will yield _empty_ results, because we still have:
https://github.com/OCR-D/core/blob/abaa6b2ad5eff549341fdc650ac2c952f7b20e2f/ocrd_utils/ocrd_utils/os.py#L74-L78

Reply:
Right, that was too hasty :/

Reply:
Please have another look, I've added a function (`are_processor_resources_directories`) that checks whether at least one of the resource parameters of a processor has `content-type == 'text/directory` and use that to set the `is_dir` flag to `list_all_resources`.

I am still not 100% happy with this but at least we now 

*  get only directories for `ocrd-eynollah-segment -L` (with https://github.com/qurator-spk/eynollah/pull/61)
* get only directories for `ocrd-tesserocr-recognize -L`, including in subdirectories
* have `ocrd-eynollah-segment -C default` print a tarball of the directory to stdout